sim/warlock/rotations.go

```diff
@@ -1,3 +1,16 @@
+/* README(2022-08):
+Main Author: Glandalf (Discord : Glandalf#0679)
+Co-Author's: Linelo aka "The good bad guy" (Discord: Linelo#3958) , Pötiküvi(Discord: Pötiküvi#7506)
```

Comment:
You should also cite Ketesh who also contributed :)

Reply:
Done! My bad!